src/server/game/Entities/AreaTrigger/AreaTriggerTemplate.h

```diff
@@ -74,6 +75,30 @@ struct AreaTriggerAction
     AreaTriggerActionUserTypes TargetType;
 };
 
+struct AreaTriggerScaleInfo
+{
+    AreaTriggerScaleInfo()
+    {
+        memset(OverrideScale, 0, sizeof(OverrideScale));
+        memset(ExtraScale, 0, sizeof(ExtraScale));
+
+        ExtraScale[5].AsFloat = 1.0000001f;
+        ExtraScale[5].AsInt32 = 1;
```

Comment:
Yes, was a typo sorry, fixed in https://github.com/TrinityCore/TrinityCore/pull/19171/commits/08bc8b58fc126329fd564a4f78748c11b63adc4f